packages/amplication-server/src/core/git/__mocks__/Github.ts

```diff
@@ -4,9 +4,9 @@ import { TEST_GIT_REPO } from "./GitRepo";
 import { TEST_GIT_REPOS } from "./GitRepos";
 
 export const MOCK_GITHUB_SERVICE = mock<GithubService>({});
-MOCK_GITHUB_SERVICE.getOrganizationRepos.mockReturnValue(
+MOCK_GITHUB_SERVICE.getRepositories.mockReturnValue(
   Promise.resolve(TEST_GIT_REPOS)
 );
-MOCK_GITHUB_SERVICE.createOrganizationRepository.mockReturnValue(
+MOCK_GITHUB_SERVICE.createRepository.mockReturnValue(
   Promise.resolve(TEST_GIT_REPO)
 );
```

Comment:
No GitHub should be needed anymore since now you can just mock the IGitClient

Reply:
yes...this is just a refactoring of what we have now. We will need to update and rewrite all our tests